crates/wdk-sys/generated_bindings/constants.rs

```diff
@@ -1958,6 +1958,7 @@ pub const PF_AVX512F_INSTRUCTIONS_AVAILABLE: u32 = 41;
 pub const PF_ERMS_AVAILABLE: u32 = 42;
```

Comment:
Diffs in the generated bindings folder are unrelated to the pr. it has to do with me using a slightly different version of WDK locally (ie. winget instead of eWDK). I am considering removing the `generated_bindings` folder from git and just feature-gating its generation for when someone actually wants to diff it.